backend/src/repos/TroopRepo.js

```diff
@@ -1,36 +1,56 @@
 import QueryHandler from './QueryHandler';
 
 export class TroopRepo extends QueryHandler {
-
-    constructor(db,errorCodes) {
-        super(db,errorCodes);
+  constructor(db, errorCodes) {
+    super(db, errorCodes);
+    this.table = {
+      name: 'kingdom_troops',
+      columns: {
+        id: 'id',
+        kingdom_id: 'kingdom_id',
+        level: 'level',
+        hp: 'hp',
+        attack: 'attack',
+        defence: 'defence',
+        started_at: 'started_at',
+        finished_at: 'finished_at',
+      },
     };
+  }
 
-    async add({kingdomId,level,hp,attack,defence,started_at,finished_at}) {
-        if (!kingdomId) throw new Error(this.errorCodes.missingKingdomId);
-        const query = this.validateQuery`
+  async add({
+    kingdom_id,
+    level,
+    hp,
+    attack,
+    defence,
+    started_at,
+    finished_at,
+  }) {
+    if (!kingdom_id) throw new Error(this.errorCodes.missingKingdomId);
+    const query = this.validateQuery`
             INSERT INTO kingdom_troops
             (kingdom_id,level,hp,attack,defence,started_at,finished_at) 
```

Comment:
@kkeszthe if you introduced `this.table.name` and `this.table.columns`, you should use those in the queries

Reply:
@adamgyulavari I tried but it breaks the SQL request. The Query handler replaces everything that is marked with ${} a ? and add them to the list of data at the end of the request. SQL was very cross with me about that.

Reply:
I don't see that in the query handler's code

Reply:
@adamgyulavari this is the one:
validateQuery(query,...params) {
        params.forEach( param => {
            if (param === undefined) throw new Error(this.errorCodes.missingParam);
        });
        return [
            query.join('?'),
            params
        ];
    };